service/src/main/java/uk/gov/hmcts/reform/fpl/service/removeorder/GeneratedOrderRemovalAction.java

```diff
@@ -29,6 +29,7 @@ public void remove(CaseData caseData,
                        RemovableOrder removableOrder) {
 
         GeneratedOrder generatedRemovableOrder = (GeneratedOrder) removableOrder;
+        ((GeneratedOrder) removableOrder).toBuilder().judgeAndLegalAdvisor(null).build();
```

Comment:
is this a new requirement?

Reply:
Yes from the attached screens orders should reset Judge object. I'd argue out of scope of this ticket but it is in the designs and was agreed I guess.